test/python/circuit/test_circuit_operations.py

```diff
@@ -1262,6 +1264,44 @@ def test_deprecated_reset_function(self):
 
         self.assertEqual(test, expected)
 
+    def test_from_instructions(self):
+        """Test from_instructions method."""
```

Comment:
It'd be good to throw in a test of the keyword arguments the method can also take too (`name`, `metadata` and `global_phase`), and potentially include a test of mixing the `CircuitInstruction` / `tuple` forms (if that's something you intend to be supported - the type hint and implementation suggest so).

Reply:
Done.